gslib/commands/ls.py

```diff
@@ -67,7 +67,7 @@
   listed provider:
 
     gsutil ls gs://
-    
+
```

Comment:
Are these spaces intentional?

Reply:
YAPF auto-formatter?
I can remove them if you want

Reply:
It's fine as long as the help text is not affected